Automattic/wp-calypso
# Calypso Settings: Align tooltip with row

On `/settings/discussion/[site-slug]`:

## Before

<img width="744" alt="image" src="https://github.com/Automattic/wp-calypso/assets/52076348/d35cae55-de10-476e-85ad-3e25c559594c">

## After

<img width="744" alt="image" src="https://github.com/Automattic/wp-calypso/assets/52076348/cd2cc6bc-9335-4403-b1ba-23ed845c6f2f">


## Testing
1. Live link
2. Visit  `/settings/discussion/[site-slug]` (should be a jetpack/atomic site)
3. Check that the tooltip are aligned


Fixes https://github.com/Automattic/wp-calypso/issues/76155
